src/models/topic-extraction/TopicExtractionJob.ts

```diff
@@ -0,0 +1,19 @@
+import { JobStatus } from '../JobStatus';
+
+/**
+ * 
+ */
+export interface TopicExtractionJob {
+    type: 'topic_extraction';
```

Comment:
why do we need this field here?
In UI we used it for the DU purpose to distinguish this job type from others